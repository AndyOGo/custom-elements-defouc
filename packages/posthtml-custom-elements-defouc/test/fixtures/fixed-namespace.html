<custom-element>content</custom-element>
<span is="builtin-element">content</span>

<namespace-element>content</namespace-element>
<span is="namespace-element">content</span>
