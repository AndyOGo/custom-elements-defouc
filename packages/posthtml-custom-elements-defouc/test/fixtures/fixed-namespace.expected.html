<custom-element>content</custom-element>
<span is="builtin-element">content</span>

<namespace-element style="visibility: hidden">content</namespace-element>
<span is="namespace-element" style="visibility: hidden">content</span>
